recipes/jarvis-tools/LICENSE

```diff
@@ -0,0 +1,13 @@
+# Terms of Use
```

Comment:
This does not seem to be the exact license file that is in the upstream repository: https://github.com/usnistgov/jarvis/blob/master/LICENSE

Is that intentional?

Reply:
No, oversight. Will fix. The upstream version is also mangled.